vim25/soap/client.go

```diff
@@ -43,48 +41,43 @@ type HasFault interface {
 }
 
 type RoundTripper interface {
-	RoundTrip(req, res HasFault) error
+	RoundTrip(ctx context.Context, req, res HasFault) error
 }
 
-var cn uint64 // Client counter
-
 type Client struct {
 	http.Client
 
-	u        url.URL
-	insecure bool
-	_        uint32 // for alignment: https://code.google.com/p/go/issues/detail?id=6404
-
-	cn  uint64         // Client counter
-	rn  uint64         // Request counter
-	log io.WriteCloser // Request log
+	u url.URL
+	k bool // Named after curl's -k flag
```

Comment:
I believe this was only needed for the atomic increments on the client counter and request counter fields (since these are 64 bit). These are now located in a separate debug container, which is aligned itself, so AFAICT this should be fine.